web/src/js/components/editors/ValueEditor.tsx

```diff
@@ -9,6 +9,7 @@ export interface ValueEditorProps {
     onInput?: (newVal: string) => void;
     onKeyDown?: (e: React.KeyboardEvent<HTMLSpanElement>) => void;
     placeholder?: string;
+    setCursorOnClick?: boolean;
```

Comment:
I'm wondering if this would be easier to understand if we'd invert it, e.g. calling it `selectAllOnClick`. WDYT?

(changing the few places where we use it shouldn't be a big deal)

Reply:
Yes, we can invert that if you want. I noticed that the `ValueEditor.tsx` component is used in many files, and its current behavior across all of them is to select the entire text on click. Where do we actually need this **"entire text selection"** feature? If it's only required in a few places, we can definitely invert the default behavior of this component.

Reply:
I removed the `selectAllOnClick` feature from the capture tab because it also includes hosts, URLs, and paths, ad it was a bit weird. What do you think?